src/atap_widgets/concordance.py

```diff
@@ -108,7 +109,7 @@ def prepare_text_df(
     df: pd.DataFrame,
     text_column: str = "text",
     id_column: str = None,
-    language_model: Union[str, spacy.language.Language] = "en_core_web_sm",
+    language_model: Union[str, spacy.language.Language] = English(),
```

Comment:
I hadn't seen this way of setting up a spacy model, and don't fully understand what it's doing relative to using `en_core_web_sm` but it does look pretty readable, so I'm happy to go with this as the default.

Can you check what components the `English` model uses and describe them in a comment or the docstring, so people know what components are necessary and which can be disabled?

Reply:
So calling `English()` is equivalent to `spacy.blank("en")`, so there are actually no components loaded. I'll add this detail in the comments